src/frontend/admin/src/components/templates/products/form/ProductForm.spec.tsx

```diff
@@ -50,8 +52,11 @@ describe("<ProductForm/>", () => {
     );
   });
 
+  afterEach(() => {
+    jest.useRealTimers();
```

Comment:
It's good to use also `jest.runOnlyPendingTimers()` before switching back to realTimers in order to ensure that all pending timers are terminated.